homeworks/elena-savitskaya/homework-3/secondScript.js

```diff
@@ -0,0 +1,10 @@
+const Number = [2, 45, 3, 67, 34, 567, 34, 345, 123]
```

Comment:
incorrect name, such variables should start from lowercase. Also name is not matched with content
```suggestion
const numberList = [2, 45, 3, 67, 34, 567, 34, 345, 123]
```